apps/docs/content/components/Pillar/react.mdx

```diff
@@ -49,6 +49,25 @@ You can add an `icon` to enhance the visual context. We recommend using an [Octi
 </Pillar>
 ```
 
+### Image
+
+You can add an `image` to enhance the visual context.
+
+```jsx live
+<Pillar>
+  <Pillar.Image
+    src="https://via.placeholder.com/600x400/d3d9df/d3d9df.png"
+    alt="placeholder, blank area with an gray background color"
+    aspectRatio="16:9"
```

Comment:
Could we keep these examples minimal for simpler copy/pasting, or do you need the aspectRatio here?

```suggestion
```

Reply:
I'm just replicating what we currently have for the Card examples. I can remove it on both to make the examples minimal.